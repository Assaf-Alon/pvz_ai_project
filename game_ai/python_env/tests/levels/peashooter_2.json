{
    "10": [["Normal", 0]],
    "20": [["Normal", 0]],
    "30": [["Normal", 0]],
    "40": [["Normal", 0]],
    "50": [["Normal", 0]],
    "60": [["Normal", 0]]
}
