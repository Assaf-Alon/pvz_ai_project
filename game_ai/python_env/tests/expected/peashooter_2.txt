[100] Normal zombie being spawned at lane 0
[100] Sun autospawned. Total: 75.
[147] Normal zombie in (0, 9) moved to (0, 8).
[194] Normal zombie in (0, 8) moved to (0, 7).
[200] Normal zombie being spawned at lane 0
[200] Sun autospawned. Total: 100.
[201] Planted Peashooter in (0, 0). Total: 0.
[216] Pea generated by Peashooter in (0, 0).
[216] Pea in (0, 0) moved to (0, 1).
[218] Pea in (0, 1) moved to (0, 2).
[220] Pea in (0, 2) moved to (0, 3).
[222] Pea in (0, 3) moved to (0, 4).
[224] Pea in (0, 4) moved to (0, 5).
[226] Pea in (0, 5) moved to (0, 6).
[228] Pea in (0, 6) moved to (0, 7).
[228] Pea in (0, 7) attacked.
[228] Normal zombie in (0, 7) was damaged. HP: 161.
[228] Pea in (0, 7) removed.
[231] Pea generated by Peashooter in (0, 0).
[231] Pea in (0, 0) moved to (0, 1).
[233] Pea in (0, 1) moved to (0, 2).
[235] Pea in (0, 2) moved to (0, 3).
[237] Pea in (0, 3) moved to (0, 4).
[239] Pea in (0, 4) moved to (0, 5).
[241] Pea in (0, 5) moved to (0, 6).
[241] Normal zombie in (0, 7) moved to (0, 6).
[242] Pea in (0, 6) attacked.
[242] Normal zombie in (0, 6) was damaged. HP: 141.
[242] Pea in (0, 6) removed.
[246] Pea generated by Peashooter in (0, 0).
[246] Pea in (0, 0) moved to (0, 1).
[247] Normal zombie in (0, 9) moved to (0, 8).
[248] Pea in (0, 1) moved to (0, 2).
[250] Pea in (0, 2) moved to (0, 3).
[252] Pea in (0, 3) moved to (0, 4).
[254] Pea in (0, 4) moved to (0, 5).
[256] Pea in (0, 5) moved to (0, 6).
[256] Pea in (0, 6) attacked.
[256] Normal zombie in (0, 6) was damaged. HP: 121.
[256] Pea in (0, 6) removed.
[261] Pea generated by Peashooter in (0, 0).
[261] Pea in (0, 0) moved to (0, 1).
[263] Pea in (0, 1) moved to (0, 2).
[265] Pea in (0, 2) moved to (0, 3).
[267] Pea in (0, 3) moved to (0, 4).
[269] Pea in (0, 4) moved to (0, 5).
[271] Pea in (0, 5) moved to (0, 6).
[271] Pea in (0, 6) attacked.
[271] Normal zombie in (0, 6) was damaged. HP: 101.
[271] Pea in (0, 6) removed.
[276] Pea generated by Peashooter in (0, 0).
[276] Pea in (0, 0) moved to (0, 1).
[278] Pea in (0, 1) moved to (0, 2).
[280] Pea in (0, 2) moved to (0, 3).
[282] Pea in (0, 3) moved to (0, 4).
[284] Pea in (0, 4) moved to (0, 5).
[286] Pea in (0, 5) moved to (0, 6).
[286] Pea in (0, 6) attacked.
[286] Normal zombie in (0, 6) was damaged. HP: 81.
[286] Pea in (0, 6) removed.
[288] Normal zombie in (0, 6) moved to (0, 5).
[291] Pea generated by Peashooter in (0, 0).
[291] Pea in (0, 0) moved to (0, 1).
[293] Pea in (0, 1) moved to (0, 2).
[294] Normal zombie in (0, 8) moved to (0, 7).
[295] Pea in (0, 2) moved to (0, 3).
[297] Pea in (0, 3) moved to (0, 4).
[299] Pea in (0, 4) moved to (0, 5).
[299] Pea in (0, 5) attacked.
[299] Normal zombie in (0, 5) was damaged. HP: 61.
[299] Pea in (0, 5) removed.
[300] Normal zombie being spawned at lane 0
[300] Sun autospawned. Total: 25.
[306] Pea generated by Peashooter in (0, 0).
[306] Pea in (0, 0) moved to (0, 1).
[308] Pea in (0, 1) moved to (0, 2).
[310] Pea in (0, 2) moved to (0, 3).
[312] Pea in (0, 3) moved to (0, 4).
[314] Pea in (0, 4) moved to (0, 5).
[314] Pea in (0, 5) attacked.
[314] Normal zombie in (0, 5) was damaged. HP: 41.
[314] Pea in (0, 5) removed.
[321] Pea generated by Peashooter in (0, 0).
[321] Pea in (0, 0) moved to (0, 1).
[323] Pea in (0, 1) moved to (0, 2).
[325] Pea in (0, 2) moved to (0, 3).
[327] Pea in (0, 3) moved to (0, 4).
[329] Pea in (0, 4) moved to (0, 5).
[329] Pea in (0, 5) attacked.
[329] Normal zombie in (0, 5) was damaged. HP: 21.
[329] Pea in (0, 5) removed.
[335] Normal zombie in (0, 5) moved to (0, 4).
[336] Pea generated by Peashooter in (0, 0).
[336] Pea in (0, 0) moved to (0, 1).
[338] Pea in (0, 1) moved to (0, 2).
[340] Pea in (0, 2) moved to (0, 3).
[341] Normal zombie in (0, 7) moved to (0, 6).
[342] Pea in (0, 3) moved to (0, 4).
[342] Pea in (0, 4) attacked.
[342] Normal zombie in (0, 4) was damaged. HP: 1.
[342] Pea in (0, 4) removed.
[347] Normal zombie in (0, 9) moved to (0, 8).
[351] Pea generated by Peashooter in (0, 0).
[351] Pea in (0, 0) moved to (0, 1).
[353] Pea in (0, 1) moved to (0, 2).
[355] Pea in (0, 2) moved to (0, 3).
[357] Pea in (0, 3) moved to (0, 4).
[357] Pea in (0, 4) attacked.
[357] Normal zombie in (0, 4) was damaged. HP: -19.
[357] Normal zombie in (0, 4) was killed.
[357] Pea in (0, 4) removed.
[366] Pea generated by Peashooter in (0, 0).
[366] Pea in (0, 0) moved to (0, 1).
[368] Pea in (0, 1) moved to (0, 2).
[370] Pea in (0, 2) moved to (0, 3).
[372] Pea in (0, 3) moved to (0, 4).
[374] Pea in (0, 4) moved to (0, 5).
[376] Pea in (0, 5) moved to (0, 6).
[376] Pea in (0, 6) attacked.
[376] Normal zombie in (0, 6) was damaged. HP: 161.
[376] Pea in (0, 6) removed.
[381] Pea generated by Peashooter in (0, 0).
[381] Pea in (0, 0) moved to (0, 1).
[383] Pea in (0, 1) moved to (0, 2).
[385] Pea in (0, 2) moved to (0, 3).
[387] Pea in (0, 3) moved to (0, 4).
[388] Normal zombie in (0, 6) moved to (0, 5).
[389] Pea in (0, 4) moved to (0, 5).
[389] Pea in (0, 5) attacked.
[389] Normal zombie in (0, 5) was damaged. HP: 141.
[389] Pea in (0, 5) removed.
[394] Normal zombie in (0, 8) moved to (0, 7).
[396] Pea generated by Peashooter in (0, 0).
[396] Pea in (0, 0) moved to (0, 1).
[398] Pea in (0, 1) moved to (0, 2).
[400] Pea in (0, 2) moved to (0, 3).
[400] Normal zombie being spawned at lane 0
[400] Sun autospawned. Total: 50.
[402] Pea in (0, 3) moved to (0, 4).
[404] Pea in (0, 4) moved to (0, 5).
[404] Pea in (0, 5) attacked.
[404] Normal zombie in (0, 5) was damaged. HP: 121.
[404] Pea in (0, 5) removed.
[411] Pea generated by Peashooter in (0, 0).
[411] Pea in (0, 0) moved to (0, 1).
[413] Pea in (0, 1) moved to (0, 2).
[415] Pea in (0, 2) moved to (0, 3).
[417] Pea in (0, 3) moved to (0, 4).
[419] Pea in (0, 4) moved to (0, 5).
[419] Pea in (0, 5) attacked.
[419] Normal zombie in (0, 5) was damaged. HP: 101.
[419] Pea in (0, 5) removed.
[426] Pea generated by Peashooter in (0, 0).
[426] Pea in (0, 0) moved to (0, 1).
[428] Pea in (0, 1) moved to (0, 2).
[430] Pea in (0, 2) moved to (0, 3).
[432] Pea in (0, 3) moved to (0, 4).
[434] Pea in (0, 4) moved to (0, 5).
[434] Pea in (0, 5) attacked.
[434] Normal zombie in (0, 5) was damaged. HP: 81.
[434] Pea in (0, 5) removed.
[435] Normal zombie in (0, 5) moved to (0, 4).
[441] Pea generated by Peashooter in (0, 0).
[441] Pea in (0, 0) moved to (0, 1).
[441] Normal zombie in (0, 7) moved to (0, 6).
[443] Pea in (0, 1) moved to (0, 2).
[445] Pea in (0, 2) moved to (0, 3).
[447] Pea in (0, 3) moved to (0, 4).
[447] Pea in (0, 4) attacked.
[447] Normal zombie in (0, 4) was damaged. HP: 61.
[447] Pea in (0, 4) removed.
[447] Normal zombie in (0, 9) moved to (0, 8).
[456] Pea generated by Peashooter in (0, 0).
[456] Pea in (0, 0) moved to (0, 1).
[458] Pea in (0, 1) moved to (0, 2).
[460] Pea in (0, 2) moved to (0, 3).
[462] Pea in (0, 3) moved to (0, 4).
[462] Pea in (0, 4) attacked.
[462] Normal zombie in (0, 4) was damaged. HP: 41.
[462] Pea in (0, 4) removed.
[471] Pea generated by Peashooter in (0, 0).
[471] Pea in (0, 0) moved to (0, 1).
[473] Pea in (0, 1) moved to (0, 2).
[475] Pea in (0, 2) moved to (0, 3).
[477] Pea in (0, 3) moved to (0, 4).
[477] Pea in (0, 4) attacked.
[477] Normal zombie in (0, 4) was damaged. HP: 21.
[477] Pea in (0, 4) removed.
[482] Normal zombie in (0, 4) moved to (0, 3).
[486] Pea generated by Peashooter in (0, 0).
[486] Pea in (0, 0) moved to (0, 1).
[488] Pea in (0, 1) moved to (0, 2).
[488] Normal zombie in (0, 6) moved to (0, 5).
[490] Pea in (0, 2) moved to (0, 3).
[490] Pea in (0, 3) attacked.
[490] Normal zombie in (0, 3) was damaged. HP: 1.
[490] Pea in (0, 3) removed.
[494] Normal zombie in (0, 8) moved to (0, 7).
[500] Normal zombie being spawned at lane 0
[500] Sun autospawned. Total: 75.
[501] Pea generated by Peashooter in (0, 0).
[501] Pea in (0, 0) moved to (0, 1).
[503] Pea in (0, 1) moved to (0, 2).
[505] Pea in (0, 2) moved to (0, 3).
[505] Pea in (0, 3) attacked.
[505] Normal zombie in (0, 3) was damaged. HP: -19.
[505] Normal zombie in (0, 3) was killed.
[505] Pea in (0, 3) removed.
[516] Pea generated by Peashooter in (0, 0).
[516] Pea in (0, 0) moved to (0, 1).
[518] Pea in (0, 1) moved to (0, 2).
[520] Pea in (0, 2) moved to (0, 3).
[522] Pea in (0, 3) moved to (0, 4).
[524] Pea in (0, 4) moved to (0, 5).
[524] Pea in (0, 5) attacked.
[524] Normal zombie in (0, 5) was damaged. HP: 161.
[524] Pea in (0, 5) removed.
[531] Pea generated by Peashooter in (0, 0).
[531] Pea in (0, 0) moved to (0, 1).
[533] Pea in (0, 1) moved to (0, 2).
[535] Pea in (0, 2) moved to (0, 3).
[535] Normal zombie in (0, 5) moved to (0, 4).
[537] Pea in (0, 3) moved to (0, 4).
[537] Pea in (0, 4) attacked.
[537] Normal zombie in (0, 4) was damaged. HP: 141.
[537] Pea in (0, 4) removed.
[541] Normal zombie in (0, 7) moved to (0, 6).
[546] Pea generated by Peashooter in (0, 0).
[546] Pea in (0, 0) moved to (0, 1).
[547] Normal zombie in (0, 9) moved to (0, 8).
[548] Pea in (0, 1) moved to (0, 2).
[550] Pea in (0, 2) moved to (0, 3).
[552] Pea in (0, 3) moved to (0, 4).
[552] Pea in (0, 4) attacked.
[552] Normal zombie in (0, 4) was damaged. HP: 121.
[552] Pea in (0, 4) removed.
[561] Pea generated by Peashooter in (0, 0).
[561] Pea in (0, 0) moved to (0, 1).
[563] Pea in (0, 1) moved to (0, 2).
[565] Pea in (0, 2) moved to (0, 3).
[567] Pea in (0, 3) moved to (0, 4).
[567] Pea in (0, 4) attacked.
[567] Normal zombie in (0, 4) was damaged. HP: 101.
[567] Pea in (0, 4) removed.
[576] Pea generated by Peashooter in (0, 0).
[576] Pea in (0, 0) moved to (0, 1).
[578] Pea in (0, 1) moved to (0, 2).
[580] Pea in (0, 2) moved to (0, 3).
[582] Pea in (0, 3) moved to (0, 4).
[582] Pea in (0, 4) attacked.
[582] Normal zombie in (0, 4) was damaged. HP: 81.
[582] Pea in (0, 4) removed.
[582] Normal zombie in (0, 4) moved to (0, 3).
[588] Normal zombie in (0, 6) moved to (0, 5).
[591] Pea generated by Peashooter in (0, 0).
[591] Pea in (0, 0) moved to (0, 1).
[593] Pea in (0, 1) moved to (0, 2).
[594] Normal zombie in (0, 8) moved to (0, 7).
[595] Pea in (0, 2) moved to (0, 3).
[595] Pea in (0, 3) attacked.
[595] Normal zombie in (0, 3) was damaged. HP: 61.
[595] Pea in (0, 3) removed.
[600] Normal zombie being spawned at lane 0
[600] Sun autospawned. Total: 100.
[601] Planted Peashooter in (0, 1). Total: 0.
[606] Pea generated by Peashooter in (0, 0).
[606] Pea in (0, 0) moved to (0, 1).
[608] Pea in (0, 1) moved to (0, 2).
[610] Pea in (0, 2) moved to (0, 3).
[610] Pea in (0, 3) attacked.
[610] Normal zombie in (0, 3) was damaged. HP: 41.
[610] Pea in (0, 3) removed.
[616] Pea generated by Peashooter in (0, 1).
[616] Pea in (0, 1) moved to (0, 2).
[618] Pea in (0, 2) moved to (0, 3).
[618] Pea in (0, 3) attacked.
[618] Normal zombie in (0, 3) was damaged. HP: 21.
[618] Pea in (0, 3) removed.
[621] Pea generated by Peashooter in (0, 0).
[621] Pea in (0, 0) moved to (0, 1).
[623] Pea in (0, 1) moved to (0, 2).
[625] Pea in (0, 2) moved to (0, 3).
[625] Pea in (0, 3) attacked.
[625] Normal zombie in (0, 3) was damaged. HP: 1.
[625] Pea in (0, 3) removed.
[629] Normal zombie in (0, 3) moved to (0, 2).
[631] Pea generated by Peashooter in (0, 1).
[631] Pea in (0, 1) moved to (0, 2).
[631] Pea in (0, 2) attacked.
[631] Normal zombie in (0, 2) was damaged. HP: -19.
[631] Normal zombie in (0, 2) was killed.
[631] Pea in (0, 2) removed.
[635] Normal zombie in (0, 5) moved to (0, 4).
[636] Pea generated by Peashooter in (0, 0).
[636] Pea in (0, 0) moved to (0, 1).
[638] Pea in (0, 1) moved to (0, 2).
[640] Pea in (0, 2) moved to (0, 3).
[641] Normal zombie in (0, 7) moved to (0, 6).
[642] Pea in (0, 3) moved to (0, 4).
[642] Pea in (0, 4) attacked.
[642] Normal zombie in (0, 4) was damaged. HP: 161.
[642] Pea in (0, 4) removed.
[646] Pea generated by Peashooter in (0, 1).
[646] Pea in (0, 1) moved to (0, 2).
[647] Normal zombie in (0, 9) moved to (0, 8).
[648] Pea in (0, 2) moved to (0, 3).
[650] Pea in (0, 3) moved to (0, 4).
[650] Pea in (0, 4) attacked.
[650] Normal zombie in (0, 4) was damaged. HP: 141.
[650] Pea in (0, 4) removed.
[651] Pea generated by Peashooter in (0, 0).
[651] Pea in (0, 0) moved to (0, 1).
[653] Pea in (0, 1) moved to (0, 2).
[655] Pea in (0, 2) moved to (0, 3).
[657] Pea in (0, 3) moved to (0, 4).
[657] Pea in (0, 4) attacked.
[657] Normal zombie in (0, 4) was damaged. HP: 121.
[657] Pea in (0, 4) removed.
[661] Pea generated by Peashooter in (0, 1).
[661] Pea in (0, 1) moved to (0, 2).
[663] Pea in (0, 2) moved to (0, 3).
[665] Pea in (0, 3) moved to (0, 4).
[665] Pea in (0, 4) attacked.
[665] Normal zombie in (0, 4) was damaged. HP: 101.
[665] Pea in (0, 4) removed.
[666] Pea generated by Peashooter in (0, 0).
[666] Pea in (0, 0) moved to (0, 1).
[668] Pea in (0, 1) moved to (0, 2).
[670] Pea in (0, 2) moved to (0, 3).
[672] Pea in (0, 3) moved to (0, 4).
[672] Pea in (0, 4) attacked.
[672] Normal zombie in (0, 4) was damaged. HP: 81.
[672] Pea in (0, 4) removed.
[676] Pea generated by Peashooter in (0, 1).
[676] Pea in (0, 1) moved to (0, 2).
[678] Pea in (0, 2) moved to (0, 3).
[680] Pea in (0, 3) moved to (0, 4).
[680] Pea in (0, 4) attacked.
[680] Normal zombie in (0, 4) was damaged. HP: 61.
[680] Pea in (0, 4) removed.
[681] Pea generated by Peashooter in (0, 0).
[681] Pea in (0, 0) moved to (0, 1).
[682] Normal zombie in (0, 4) moved to (0, 3).
[683] Pea in (0, 1) moved to (0, 2).
[685] Pea in (0, 2) moved to (0, 3).
[685] Pea in (0, 3) attacked.
[685] Normal zombie in (0, 3) was damaged. HP: 41.
[685] Pea in (0, 3) removed.
[688] Normal zombie in (0, 6) moved to (0, 5).
[691] Pea generated by Peashooter in (0, 1).
[691] Pea in (0, 1) moved to (0, 2).
[693] Pea in (0, 2) moved to (0, 3).
[693] Pea in (0, 3) attacked.
[693] Normal zombie in (0, 3) was damaged. HP: 21.
[693] Pea in (0, 3) removed.
[694] Normal zombie in (0, 8) moved to (0, 7).
[696] Pea generated by Peashooter in (0, 0).
[696] Pea in (0, 0) moved to (0, 1).
[698] Pea in (0, 1) moved to (0, 2).
[700] Pea in (0, 2) moved to (0, 3).
[700] Pea in (0, 3) attacked.
[700] Normal zombie in (0, 3) was damaged. HP: 1.
[700] Pea in (0, 3) removed.
[700] Sun autospawned. Total: 25.
[706] Pea generated by Peashooter in (0, 1).
[706] Pea in (0, 1) moved to (0, 2).
[708] Pea in (0, 2) moved to (0, 3).
[708] Pea in (0, 3) attacked.
[708] Normal zombie in (0, 3) was damaged. HP: -19.
[708] Normal zombie in (0, 3) was killed.
[708] Pea in (0, 3) removed.
[711] Pea generated by Peashooter in (0, 0).
[711] Pea in (0, 0) moved to (0, 1).
[713] Pea in (0, 1) moved to (0, 2).
[715] Pea in (0, 2) moved to (0, 3).
[717] Pea in (0, 3) moved to (0, 4).
[719] Pea in (0, 4) moved to (0, 5).
[719] Pea in (0, 5) attacked.
[719] Normal zombie in (0, 5) was damaged. HP: 161.
[719] Pea in (0, 5) removed.
[721] Pea generated by Peashooter in (0, 1).
[721] Pea in (0, 1) moved to (0, 2).
[723] Pea in (0, 2) moved to (0, 3).
[725] Pea in (0, 3) moved to (0, 4).
[726] Pea generated by Peashooter in (0, 0).
[726] Pea in (0, 0) moved to (0, 1).
[727] Pea in (0, 4) moved to (0, 5).
[727] Pea in (0, 5) attacked.
[727] Normal zombie in (0, 5) was damaged. HP: 141.
[727] Pea in (0, 5) removed.
[728] Pea in (0, 1) moved to (0, 2).
[730] Pea in (0, 2) moved to (0, 3).
[732] Pea in (0, 3) moved to (0, 4).
[734] Pea in (0, 4) moved to (0, 5).
[734] Pea in (0, 5) attacked.
[734] Normal zombie in (0, 5) was damaged. HP: 121.
[734] Pea in (0, 5) removed.
[735] Normal zombie in (0, 5) moved to (0, 4).
[736] Pea generated by Peashooter in (0, 1).
[736] Pea in (0, 1) moved to (0, 2).
[738] Pea in (0, 2) moved to (0, 3).
[740] Pea in (0, 3) moved to (0, 4).
[740] Pea in (0, 4) attacked.
[740] Normal zombie in (0, 4) was damaged. HP: 101.
[740] Pea in (0, 4) removed.
[741] Pea generated by Peashooter in (0, 0).
[741] Pea in (0, 0) moved to (0, 1).
[741] Normal zombie in (0, 7) moved to (0, 6).
[743] Pea in (0, 1) moved to (0, 2).
[745] Pea in (0, 2) moved to (0, 3).
[747] Pea in (0, 3) moved to (0, 4).
[747] Pea in (0, 4) attacked.
[747] Normal zombie in (0, 4) was damaged. HP: 81.
[747] Pea in (0, 4) removed.
[751] Pea generated by Peashooter in (0, 1).
[751] Pea in (0, 1) moved to (0, 2).
[753] Pea in (0, 2) moved to (0, 3).
[755] Pea in (0, 3) moved to (0, 4).
[755] Pea in (0, 4) attacked.
[755] Normal zombie in (0, 4) was damaged. HP: 61.
[755] Pea in (0, 4) removed.
[756] Pea generated by Peashooter in (0, 0).
[756] Pea in (0, 0) moved to (0, 1).
[758] Pea in (0, 1) moved to (0, 2).
[760] Pea in (0, 2) moved to (0, 3).
[762] Pea in (0, 3) moved to (0, 4).
[762] Pea in (0, 4) attacked.
[762] Normal zombie in (0, 4) was damaged. HP: 41.
[762] Pea in (0, 4) removed.
[766] Pea generated by Peashooter in (0, 1).
[766] Pea in (0, 1) moved to (0, 2).
[768] Pea in (0, 2) moved to (0, 3).
[770] Pea in (0, 3) moved to (0, 4).
[770] Pea in (0, 4) attacked.
[770] Normal zombie in (0, 4) was damaged. HP: 21.
[770] Pea in (0, 4) removed.
[771] Pea generated by Peashooter in (0, 0).
[771] Pea in (0, 0) moved to (0, 1).
[773] Pea in (0, 1) moved to (0, 2).
[775] Pea in (0, 2) moved to (0, 3).
[777] Pea in (0, 3) moved to (0, 4).
[777] Pea in (0, 4) attacked.
[777] Normal zombie in (0, 4) was damaged. HP: 1.
[777] Pea in (0, 4) removed.
[781] Pea generated by Peashooter in (0, 1).
[781] Pea in (0, 1) moved to (0, 2).
[782] Normal zombie in (0, 4) moved to (0, 3).
[783] Pea in (0, 2) moved to (0, 3).
[783] Pea in (0, 3) attacked.
[783] Normal zombie in (0, 3) was damaged. HP: -19.
[783] Normal zombie in (0, 3) was killed.
[783] Pea in (0, 3) removed.
[786] Pea generated by Peashooter in (0, 0).
[786] Pea in (0, 0) moved to (0, 1).
[788] Pea in (0, 1) moved to (0, 2).
[788] Normal zombie in (0, 6) moved to (0, 5).
[790] Pea in (0, 2) moved to (0, 3).
[792] Pea in (0, 3) moved to (0, 4).
[794] Pea in (0, 4) moved to (0, 5).
[794] Pea in (0, 5) attacked.
[794] Normal zombie in (0, 5) was damaged. HP: 161.
[794] Pea in (0, 5) removed.
[796] Pea generated by Peashooter in (0, 1).
[796] Pea in (0, 1) moved to (0, 2).
[798] Pea in (0, 2) moved to (0, 3).
[800] Pea in (0, 3) moved to (0, 4).
[800] Sun autospawned. Total: 50.
[801] Pea generated by Peashooter in (0, 0).
[801] Pea in (0, 0) moved to (0, 1).
[802] Pea in (0, 4) moved to (0, 5).
[802] Pea in (0, 5) attacked.
[802] Normal zombie in (0, 5) was damaged. HP: 141.
[802] Pea in (0, 5) removed.
[803] Pea in (0, 1) moved to (0, 2).
[805] Pea in (0, 2) moved to (0, 3).
[807] Pea in (0, 3) moved to (0, 4).
[809] Pea in (0, 4) moved to (0, 5).
[809] Pea in (0, 5) attacked.
[809] Normal zombie in (0, 5) was damaged. HP: 121.
[809] Pea in (0, 5) removed.
[811] Pea generated by Peashooter in (0, 1).
[811] Pea in (0, 1) moved to (0, 2).
[813] Pea in (0, 2) moved to (0, 3).
[815] Pea in (0, 3) moved to (0, 4).
[816] Pea generated by Peashooter in (0, 0).
[816] Pea in (0, 0) moved to (0, 1).
[817] Pea in (0, 4) moved to (0, 5).
[817] Pea in (0, 5) attacked.
[817] Normal zombie in (0, 5) was damaged. HP: 101.
[817] Pea in (0, 5) removed.
[818] Pea in (0, 1) moved to (0, 2).
[820] Pea in (0, 2) moved to (0, 3).
[822] Pea in (0, 3) moved to (0, 4).
[824] Pea in (0, 4) moved to (0, 5).
[824] Pea in (0, 5) attacked.
[824] Normal zombie in (0, 5) was damaged. HP: 81.
[824] Pea in (0, 5) removed.
[826] Pea generated by Peashooter in (0, 1).
[826] Pea in (0, 1) moved to (0, 2).
[828] Pea in (0, 2) moved to (0, 3).
[830] Pea in (0, 3) moved to (0, 4).
[831] Pea generated by Peashooter in (0, 0).
[831] Pea in (0, 0) moved to (0, 1).
[832] Pea in (0, 4) moved to (0, 5).
[832] Pea in (0, 5) attacked.
[832] Normal zombie in (0, 5) was damaged. HP: 61.
[832] Pea in (0, 5) removed.
[833] Pea in (0, 1) moved to (0, 2).
[835] Pea in (0, 2) moved to (0, 3).
[835] Normal zombie in (0, 5) moved to (0, 4).
[837] Pea in (0, 3) moved to (0, 4).
[837] Pea in (0, 4) attacked.
[837] Normal zombie in (0, 4) was damaged. HP: 41.
[837] Pea in (0, 4) removed.
[841] Pea generated by Peashooter in (0, 1).
[841] Pea in (0, 1) moved to (0, 2).
[843] Pea in (0, 2) moved to (0, 3).
[845] Pea in (0, 3) moved to (0, 4).
[845] Pea in (0, 4) attacked.
[845] Normal zombie in (0, 4) was damaged. HP: 21.
[845] Pea in (0, 4) removed.
[846] Pea generated by Peashooter in (0, 0).
[846] Pea in (0, 0) moved to (0, 1).
[848] Pea in (0, 1) moved to (0, 2).
[850] Pea in (0, 2) moved to (0, 3).
[852] Pea in (0, 3) moved to (0, 4).
[852] Pea in (0, 4) attacked.
[852] Normal zombie in (0, 4) was damaged. HP: 1.
[852] Pea in (0, 4) removed.
[856] Pea generated by Peashooter in (0, 1).
[856] Pea in (0, 1) moved to (0, 2).
[858] Pea in (0, 2) moved to (0, 3).
[860] Pea in (0, 3) moved to (0, 4).
[860] Pea in (0, 4) attacked.
[860] Normal zombie in (0, 4) was damaged. HP: -19.
[860] Normal zombie in (0, 4) was killed.
[860] Pea in (0, 4) removed.
[860] You've won!
